scripts/make_src_archive.sh

```diff
@@ -5,7 +5,11 @@ set -eu
 REPO=$1
 OUTPUT=$2
 
-pip3 install --user "https://files.pythonhosted.org/packages/60/44/26791beafa5c2c1f8ac0dd38a7761a0162c0f31bdad91a0091a48a923a44/git_archive_all-1.19.4-py2.py3-none-any.whl"
+python3 -m venv venv
+
+. venv/bin/activate
+
+pip3 install "https://files.pythonhosted.org/packages/60/44/26791beafa5c2c1f8ac0dd38a7761a0162c0f31bdad91a0091a48a923a44/git_archive_all-1.19.4-py2.py3-none-any.whl"
```

Comment:
Referring to a specific package/wheel version might cause an issue once the package is removed from repo.

Reply:
Pypi strongly discourages removing released packages, so it shouldn't really happen. The bigger risk would be that the pypi account of the maintainer is compromised and serve a malicious new version.

I didn't add a hash check though, so I could add one or maybe I might as well do `pip install 'git_archive_all==1.19.4'`.

Reply:
Yeah, but I experienced it at the previous job :). Will install 'git_archive_all>=1.19.4' work ?

Reply:
That's exactly what's vulnerable against a compromise.

Was it also a package from PyPi that went missing?

Reply:
Yeah, you're right from hacking/vulnerability standpoint of view.
Yes, but in the case I mentioned we actually had our own repo/PyPi server and we couldn't keep all versions there for forever and were cleaning older packages time to time, I guess official PyPi/pythonhosted won't keep all versions for forever either... 
Anyway, it's not really an issue, the version can be updated once the current doesn't work.

Reply:
Ah I see. Well I think that PyPi intends to keep every version by design, so that nothing breaks in the future. Though I haven't found that spelled out explicitly after a quick search. Some repositories, like cargo are even more strict and don't even allow the publishers to remove an existing version.

I'll maybe change it to the simpler install syntax, after I've verified that this actually works end-to-end (waiting for a bitbake on CI right now...)